contracts/NodeVersionBeacon.cdc

```diff
@@ -116,94 +120,133 @@ pub contract ExecutionNodeVersionBeacon {
     /// Sorted Array containing historical block heights where version boundaries were defined
     access(contract) var archivedBlockBoundaries: [UInt64]
 
-    /// Admin resource that manages the Execution Node versioning
+    /// Sequence number of the NodeVersionTableUpdated event
+    access(contract) var nextTableUpdatedSequence: UInt64
+
+    /// Admin resource that manages node versioning
     /// maintained in this contract
-    pub resource ExecutionNodeVersionAdmin {
+    pub resource NodeVersionAdmin {
+
         /// Update the minimum version to take effect at the given block height
         pub fun addVersionBoundaryToTable(targetBlockHeight: UInt64, newVersion: Semver) {
             pre {
-                targetBlockHeight > getCurrentBlock().height + ExecutionNodeVersionBeacon.versionUpdateBuffer
+                targetBlockHeight > getCurrentBlock().height + NodeVersionBeacon.versionUpdateBuffer
                     : "Target block height occurs too soon to update EN version."
             }
 
             // Insert mapping into version table
-            ExecutionNodeVersionBeacon.versionTable.insert(
+            NodeVersionBeacon.versionTable.insert(
                 key: targetBlockHeight,
                 newVersion
             )
             // Insert the version boundary's target block height into the
             // array maintaining all upcoming block height boundaries
-            ExecutionNodeVersionBeacon.insertUpcomingBlockBoundary(targetBlockHeight)
+            NodeVersionBeacon.insertUpcomingBlockBoundary(targetBlockHeight)
 
-            emit ExecutionNodeVersionTableAddition(height: targetBlockHeight, version: newVersion)
+            //emit NodeVersionTableAddition(height: targetBlockHeight, version: newVersion)
```

Comment:
Why is this commented?

Reply:
It should be deleted, the goal of this is to have a function that can be call just to emit the event rather than having an event emitted every time the table is updated.